src/ui/__tests__/ui_i18n_keys.js

```diff
@@ -0,0 +1,50 @@
+import expect from 'expect.js';
```

Comment:
That is ok. I just added the UTs to show how it can be called until I understood how you wanted it called.